interfaces/cython/setup-debian.py

```diff
@@ -0,0 +1,7 @@
+# This script is only used with on Debian based systems with the patched
+# versions of patched versions of setuptools that are installed there.
```

Comment:
patched versions of patched versions ... :joy:

Reply:
Well, it's probably true, but yes. 😂

Reply:
honestly thought this was intentional ...